### DISCUSSION TITLE
How to filter by enums withou namespacing

### DISCUSSION THREAD

**Discussion by petrkasnal** [OData/AspNetCoreOData#953] [Q&A] [answered]
Hi, is it possible filter my data by enum value without use fullnamespace in url? 

Thank you

**Comment by habbes** [comment]
I'm not sure if this is supported. Any ideas @KenitoInc, @xuzhg , @ElizabethOkerio, @gathogojr ?

**Comment by KenitoInc** [comment, marked as answer]
This is supported in odata core.
We have a `StringAsEnumResolver` https://github.com/OData/odata.net/blob/master/src/Microsoft.OData.Core/UriParser/Resolver/StringAsEnumResolver.cs

Look at this test https://github.com/OData/odata.net/blob/4414000c0da72b22ff3c4a96f51cbf35f2316bca/test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriParser/EnumFilterFunctionalTests.cs#L263
